.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Extractors/ChineseTimePeriodExtractorConfiguration.cs

```diff
@@ -1,59 +1,58 @@
-﻿using System.Collections.Generic;

```

Comment:
Be careful with unintentional line ending changes. They just make reviewing harder.